code/modules/mob/new_player/new_player.dm

```diff
@@ -185,9 +185,14 @@
 				to_chat(src, "<span class='notice'>As you observed [period_human_readable], you can freely toggle antag-hud without losing respawnability.</span>")
 			observer.started_as_observer = 1
 			close_spawn_windows()
-			var/obj/O = locate("landmark*Observer-Start")
+			var/obj/spawn_point
+			if(SSticker.current_state < GAME_STATE_PLAYING)
+				spawn_point = pick(GLOB.roundstart_observer_start)
+			else
+				spawn_point = locate("landmark*Observer-Start")
```

Comment:
God we need to obliterate all the tag usage soon

Reply:
yeah I wasn't entirely sure what was going on here and I don't quite like it